src/pylutron_caseta/leap.py

```diff
@@ -84,7 +85,7 @@ async def run(self):
             if received == b"":
                 break
 
-            resp_json = json.loads(received.decode("UTF-8"))
+            resp_json = orjson.loads(received.decode("UTF-8"))
```

Comment:
The decode here should not be needed as `orjson.loads` accepts `bytes`

Reply:
https://github.com/ijl/orjson#deserialize

Reply:
> bytes, bytearray, memoryview, and str input are accepted. If the input exists as a memoryview, bytearray, or bytes object, it is recommended to pass these directly rather than creating an unnecessary str object. That is, orjson.loads(b"{}") instead of orjson.loads(b"{}".decode("utf-8")). This has lower memory usage and lower latency.

Reply:
👍